be/src/service/internal_service.cpp

```diff
@@ -2054,4 +2055,11 @@ void PInternalService::get_wal_queue_size(google::protobuf::RpcController* contr
     }
 }
 
+void PInternalServiceImpl::alter_vault_sync(google::protobuf::RpcController* controller,
+                                            const doris::PAlterVaultSyncRequest* request,
+                                            PAlterVaultSyncResponse* response,
+                                            google::protobuf::Closure* done) {
+    _engine.to_cloud().sync_storage_vault();
```

Comment:
Implement this method in `CloudBackendService`, `to_cloud()` here will check failed